salt/salt.spec

```
@@ -1306,6 +1313,12 @@ systemd-tmpfiles --create /usr/lib/tmpfiles.d/salt.conf || true
 dbus-uuidgen --ensure
 %endif
 
+%if 0%{?suse_version} > 1500
+%posttrans
+# Migration to /usr/etc, restore just created .rpmsave
+test -f %{_sysconfdir}/logrotate.d/salt.rpmsave && mv -v %{_sysconfdir}/logrotate.d/salt.rpmsave %{_sysconfdir}/logrotate.d/salt ||:
```

For me it looks like we need to put `salt.rpmsave.old` made in `%pre` back to it's original place, so I would suggest to do the following:
```suggestion
test -f %{_sysconfdir}/logrotate.d/salt.rpmsave && mv -v %{_sysconfdir}/logrotate.d/salt.rpmsave %{_sysconfdir}/logrotate.d/salt ||:
test -f %{_sysconfdir}/logrotate.d/salt.rpmsave.old && mv -v %{_sysconfdir}/logrotate.d/salt.rpmsave.old %{_sysconfdir}/logrotate.d/salt.rpmsave ||:
```